lib/PicoDb/Driver/Mssql.php

```diff
@@ -175,4 +175,26 @@ public function explain($sql, array $values)
         $this->getConnection()->exec('SET SHOWPLAN_ALL ON');
         return $this->getConnection()->query($this->getSqlFromPreparedStatement($sql, $values))->fetchAll(PDO::FETCH_ASSOC);
     }
+
+    /**
+     * Get offset limit clause
+     *
+     * @param int $limit
+     * @param int $offset
+     * @return string
+     */
+    public function getLimitClause($limit, $offset)
+    {
+        $clause = '';
+
+        if (! is_null($limit)) {
+            $clause .= ' FETCH NEXT '.$limit.' ROWS ONLY';
+        }
+
+        if (! is_null($offset)) {
+            $clause = ' OFFSET '.$offset.' ROWS';
```

Comment:
```suggestion
            $clause .= ' OFFSET '.$offset.' ROWS';
```

Reply:
Actually, does OFFSET need to come first for mssql?

Reply:
Good catch! I've switched them, but left the concat operator out since it's the first thing added.